lightkurve/correctors/pldcorrector.py

```diff
@@ -85,52 +88,158 @@ class PLDCorrector(Corrector):
-            A boolean array describing the aperture such that `True` means
-            that the pixel will be used to generate the raw flux light curve.
-            If `None` or 'all' are passed, all pixels will be used.
-            If 'pipeline' is passed, the mask suggested by the official pipeline
-            will be returned.
-            If 'threshold' is passed, all pixels brighter than 3-sigma above
-            the median flux will be used.
-        cadence_mask : array-like
-            A mask that will be applied to the cadences prior to constructing
-            the detrending model. For example, you can pass a boolean array
-            of length `n_cadences` where `True` means that the cadence will be
-            included in the noise model. You may also pass an array of indices.
-            This option enables signals of interest (e.g. planet transits)
-            to be excluded from the noise model, which will prevent over-fitting.
-            By default, no cadences will be masked.
-        gp_timescale : float
-            Gaussian Process time scale length term (`tau`) used to define
-            length of fit variability in days.
-        use_gp : boolean
-            Option to turn GP fitting on or off.  You would typically only set
-            this to False to speed up the correction (at the cost of precision),
-            or if you suspect the presence of systematic noise at long timescales.
         pld_order : int
             The order of Pixel Level De-correlation to be performed. First order
             (`n=1`) uses only the pixel fluxes to construct the design matrix.
             Higher order populates the design matrix with columns constructed
             from the products of pixel fluxes.
+        pixel_components : int
+            Number of principal components derived from the background pixel
+            time series to utilize.
+        background_mask : array-like or None
+            A boolean array flagging the background pixels such that `True` means
+            that the pixel will be used to generate the background systematics model.
+            If `None`, all pixels which are fainter than 1-sigma above the median
+            flux will be used.
+        pld_aperture_mask : array-like, 'pipeline', 'all', 'threshold', or None
+            A boolean array describing the aperture such that `True` means
+            that the pixel will be used when selecting the PLD basis vectors.
+            If `None` or `all` are passed in, all pixels will be used.
+            If 'pipeline' is passed, the mask suggested by the official pipeline
+            will be returned.
+            If 'threshold' is passed, all pixels brighter than 3-sigma above
+            the median flux will be used.
+        spline_n_knots : int
+            Number of knots in spline.
+        spline_degree : int
+            Polynomial degree of spline.
         n_pca_terms : int
```

Comment:
`n_pca_terms` doesn't appear to be used at all, we're using the `pixel_components` argument instead, so all references to `n_pca_terms` should probably go.